db/controllers/articles.controllers.js

```diff
@@ -0,0 +1,12 @@
+const { fetchArticle } = require("../models/articles.models");
+
+exports.getArticle = (req, res, next) => {
+  const articleId = req.params.articleId;
+  return fetchArticle(articleId)
+    .then((article) => {
+      res.status(200).send(article);
```

Comment:
Any data sent from a controller needs to be sent back on a meaningful key ie. .send({article})